### DISCUSSION TITLE
Adding new language not supported

### DISCUSSION THREAD

**Discussion by djimdba** [openai/whisper#291] [Q&A]
How can I train the model to take into account an unsupported language?

**Comment by stefangrotz** [comment]
I would love to know this as well! A few unsupported language work quite well, at least with the "to english" translation feature. I bet they have trained the model with more languages than officially supported. 
The only obvious thing to edit for me is the file [tokenizer.py](https://github.com/openai/whisper/blob/main/whisper/tokenizer.py)
But adding a language code there surely isn't enough, especially if you want to retrain the model.

**Comment by danielinux7** [comment]
Hello,
Check out this post:
https://github.com/openai/whisper/discussions/349
